packages/fxa-auth-server/lib/metrics/glean/index.ts

```diff
@@ -104,7 +111,7 @@ const createEventFn =
       await gleanEventLogger.record(metrics);
     };
 
-export const gleanMetrics = (config: ConfigType) => {
+export function gleanMetrics(config: ConfigType) {
```

Comment:
`gleanMetrics` is used above in a type definition.  And even though `tsc` didn't have a problem with it, the linter did not like it.